repos/rust-lang/impl-trait-utils.toml

```
@@ -7,6 +7,8 @@ bots = []
 wg-async = "write"
 
 [access.individuals]
+compiler-errors = "maintain"
+TC = "maintain"
```

```suggestion
traviscross = "maintain"
```

The tools here seem to work based off of the GH username.